micro-rdk/src/common/webrtc/ice.rs

```diff
@@ -193,6 +192,7 @@ impl ICEAgent {
                     Err(io::Error::new(io::ErrorKind::TimedOut, ""))
                 })
                 .await;
+
```

Comment:
This is just ws noise from when I had some logging in this file, I'll revert it.